### ISSUE TITLE
Missing invalid plugin name header check in Readme Check

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#397]
This is the check implemented in wordpress.org readme validator. It is missing in our Readme Check.

In Readme check, following check is 
```php
// Fatal errors.
if ( empty( $readme->name ) || isset( $readme->warnings['invalid_plugin_name_header'] ) ) {
	$errors[] = sprintf(
		/* translators: 1: 'Plugin Name' section title, 2: 'Plugin Name' */
		__( 'We cannot find a plugin name in your readme. Plugin names look like: %1$s. Please change %2$s to reflect the actual name of your plugin.', 'wporg-plugins' ),
		'<code>=== Plugin Name ===</code>',
		'<code>Plugin Name</code>'
	);
}
```

Ref: https://github.com/WordPress/wordpress.org/blob/trunk/wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-validator.php#L73
